cmd/kosli/expectDeployment.go

```diff
@@ -98,7 +98,7 @@ func (o *expectDeploymentOptions) run(args []string) error {
 	}
 	_, err = kosliClient.Do(reqParams)
 	if err == nil && !global.DryRun {
-		logger.Info("deployment of artifact %s was reported to: %s", o.payload.Fingerprint, o.payload.Environment)
+		logger.Info("expect deployment of artifact %s was reported to: %s", o.payload.Fingerprint, o.payload.Environment)
```

Comment:
Theres's a couple of tests failing after this: https://github.com/kosli-dev/cli/actions/runs/5690677651/job/15424492821